lib/src/transaction.rs

```diff
@@ -166,51 +167,24 @@ impl UnpublishedOperation {
         view: View,
         index: Box<dyn ReadonlyIndex>,
     ) -> Self {
-        let data = Some(NewRepoData {
-            operation,
-            view,
-            index,
-        });
         UnpublishedOperation {
-            repo_loader,
-            data,
-            closed: false,
+            op_heads_store: repo_loader.op_heads_store().clone(),
```

Comment:
nit: pass `repo_loader` to `new()` by reference? (though it doesn't practically matter.)

Reply:
Yeah, that makes sense. Done!